src/services/performanceDbApi.js

```diff
@@ -216,7 +216,7 @@ export default {
     }
 
     let message
-    if (entityCount && numIssues >= entityCount) {
+    if (rowCount && numIssues >= rowCount) {
```

Comment:
Please update the bare `rowCount` to an explicit test (`Number.isInteger(...)` or `rowCount > 0`). Otherwise, if `rowCount = 0`, this test will fail, and it may not be obvious it should.